City-of-Helsinki/drupal-asuntotuotanto
# SATT-39: 39: Apartment status color fixes

### Description

Fixes previous SATT-39 apartment status color issues


### How to test it

- make fresh or make drush-cr
- You need latest or production sql dump
- check that project https://asuntotuotanto.docker.so/fi/asuntohaku/hitas/as-oy-helsingin-ahdinlaituri-lansisatamankatu-41 apartment statuses looking good so free is green and reserved is red
